src/components/Board/Board.css

```diff
@@ -0,0 +1,39 @@
+.board {
+  height: 540px;
+
+  width: 630px;
+
+  background-color: blue;
+  border: 10px solid navy;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  /* cursor: pointer; */
+
+  /* cursor: url("../../images/actual-red-disc.png"), auto; */
+  cursor: url("../../images/actual-yellow-disc.png"), auto;
```

Comment:
You should always alphabetise your CSS. Also, don't leave commented out code in your files:

```
  background-color: blue;
  border: 10px solid navy; 
  cursor: url("../../images/actual-yellow-disc.png"), auto;
  display: flex;
  flex-wrap: wrap;
  height: 540px;
  margin: 0 auto;
  width: 630px;
```

Reply:
Thank you, I'll adjust it.